spec/support/sequel_db.rb

```diff
@@ -0,0 +1,10 @@
+def sequel_db
+  @sequel_db = Sequel.connect(if defined?(JRUBY_VERSION)
```

Comment:
Similar to `active_record_schema`: put the Sequel database connection in its own method so that we can easily create new Sequel models in subsequent files.

This is in preparation to increase the amount of Sequel testing.